DQM/SiTrackerPhase2/plugins/BuildFile.xml

```diff
@@ -0,0 +1,14 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/PluginManager"/>
+<use name="FWCore/ParameterSet"/>
+<use name="DataFormats/TrackReco"/>
+<use   name="SimDataFormats/TrackerDigiSimLink"/>
+<use   name="DataFormats/SiPixelDigi"/>
+<use   name="DataFormats/SiPixelDetId"/>
+<use   name="DataFormats/DetId"/>
+<use   name="Geometry/Records"/>
+<use   name="Geometry/CommonDetUnit"/>
+<use   name="Geometry/TrackerGeometryBuilder"/>
+<use   name="DQMServices/Core"/>
+<use   name="DataFormats/Common"/>
+<use   name="SimTracker/SiPhase2Digitizer"/>
```

Comment:
@gsaha009 why is the plugin declaration missing?

Reply:
I have checked and updated the buildfiles. Please ckeck